modules/apps/liferay-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,115 @@
+
+body {
+	font-family: Helvetica, Arial, sans-serif;
+	margin: 0;
+}
+
+.sign-in {
+	border: 2px solid #000;
+	border-radius: 200px;
+	color: #000;
+	display: block;
+	float: right;
+	margin-right: 15px;
+	padding: 6px;
+	text-transform: uppercase;
+}
+
+.header-banner {
+	margin-bottom: 80px;
+	padding-top: 90px;
+}
+
+.logo-image {
+	display: block;
+	
+	img {
+		display: block;
+		height: auto;
+		margin: 0 auto;
+		margin-bottom: 20px;
+		width: 25%;
+	}
+}
+
+h1 {
+	font-size: 2.5em;
+	letter-spacing: 10px;
+	text-align: center;
+	text-transform: uppercase;
+} 
+
+.beginning {
+	padding: 10px;
+	text-align: center;
+}
+
+.navigation-bar {
+	align-items: center;
+	display: flex;
+	flex-direction: row;
+	height: 40px;
+	justify-content: center;
+	margin-top: 100px;
+	z-index: 1;
+
+	ul {
+		display: flex;
+		flex-wrap: wrap;
+		position: absolute;
```

Comment:
We should probably scope this to only the sub children in the navigation menu since this affects the parent level navigation items as well.